content/en/docs/concepts/cluster-administration/system-traces.md

```diff
@@ -103,7 +103,7 @@ tracing:
 If the `samplingRatePerMillion` is set to one million (`1000000`), then every
 span will be sent to the exporter.
 
-The kubelet in Kubernetes v{{< skew currentVersion >}} collects spans from
+The kubelet in Kubernetes v{{< skew currentPatchVersion >}} collects spans from
```

Comment:
The original is fine; this statement applies to all patch versions in a minor release.